behaviors/v1_1/management-chain.md

```diff
@@ -0,0 +1,237 @@
+
+## Virtual Chain Flows:
+
+### RequestOrderedCommittee flow (Read)
+* At the beginning of each consensus round (block height), the node queries the `ConsensusContext` for a sorted list of nodes (committee).
+* The `ConsensusContext` retrieves the current Elected Validators Set from state.
+    * Poll until success (retry on error).
+        * Get the Elected Validators state record  by calling `_ElectedValidators.getElectedValidators()` (using the `VM.CallSystemContract`).
+        * If the Elected Validators state record is empty (state update has yet to occur) the `ConsensusContext`  the `_ElectedValidators` will return the Reference Validator Set by calling `_Provision.getReferenceElectedValidators()` (currently not supported).
+            * Note: the Consumer-VC will query the Management-VC `_Provision` system contract. 
+    * If the resulting Elected Validator record is empty (with no error) get the Reference Validators Set from local config file.
+    * Note: empty set with no error implies "Local Reference Validators" mode.
+* The `ConsensusContext` retrieves the ordered committee by calling `_Committee.getOrderedCommitteeForAddresses` with the Elected Validators Set as argument.
+* Note: Management-VC contract call is detailed in the update flow.
+* Note: The Management-VC, Consumer-VC and Private-VC behaviors coincide here (abstracted away by the Elected Provider system contract).
+
+#### Participant Services
+##### Consumer-VC
+* `ConsensusAlgo` - ConsensusAlgo requests committee for new consensus round.
+* `ConsensusContext` - Retrieves the ordered committee based on state.
+* `VirtualMachine` - Provides the `ConsensusContext` with state queries access.
+* `CrossChainConnector` - Optional - Get Reference elected validators from Management-VC.
+
+##### Management-VC - Optional (reference set remote)
+* Detailed in update flow.
+
+
+### Consumer-VC block creation \ validation flow
+* If the elected validators set is outdated the block should be empty - with only a single system transaction (`Trigger tx`).
+    * Enforced by using the pre order checks - `_GlobalPreOrder` contract which determines whether the current elected validators record is outdated.
+    * See [Block creation flow](../flows/block-creation-before-commit.md) for further details.
+
+
+### Consumer-VC update elected validators flow
+* `ConsensusAlgo` calls the `ConsensusContext` to create \ validate a new block proposal.
+* `ConsensusContext` calls the VM to execute the block transactions using a proposer \ validator consensus role accordingly.
+* The `VirtualMachine` executes the `Trigger transaction` which in turn calls the `_ElectedValidators.sync()` method.
+* If the elected validators set should be updated the sync method retrieves an updated record by using the SDK call `Mgmt.CallContract('_ElectedValidators', 'getElectedValidators')`. 
+* `Mgmt.CallContract` acquires the new record using two consecutive queries to the Management-VC:
```

Comment:
two calls add out-of-process latency (~10ms), this is an optimization but it can be avoided, unless I'm missing something, by passing the time to the second call, thus eliminating the first.